doc/examle_ospd_log.conf

```
@@ -0,0 +1,50 @@
+### If a custom logging configuration is required, the following
+### items under General must be present in the configuration file.
+### Examples for customization are given. Just comment/uncomment
+### the corresponding lines and do necessary adjustments.
+
+## General
+####################
+
+[loggers]
+keys=root
+#keys=root, ospd_openvas
+
+[logger_root]
+level=NOTSET
+handlers=default_handler
+
+
+### There is already an existen default_handler.
+### Uncomment the following to extend the existent handler list
+####################
+
+#[handlers]
+#keys=default_handler, custom_syslog
+
+
+### Example for a custom handler. Custom must be added to the handlers list,
+####################
+
+#[handler_custom]
+#class=FileHandler
+#level=DEBUG
+#formatter=file
+#args=('some_path_to_log_file.log', 'a')
+
+#[handler_custom_syslog]
+#class=handlers.SysLogHandler
+#level=DEBUG
+#formatter=file
+#args=('/dev/log', handlers.SysLogHandler.LOG_USER)
+
+
+### Specific logging configuration for a single module. In the following
+### example, the ospd_openvas.lock module will log with debug level.
+####################
+
+#[logger_ospd_openvas]
+#level=DEBUG
+#handlers=custom_syslog
+#qualname=ospd_openvas.lock
+#propagate=0
```

Newline at the end of the file was missing
```suggestion
#propagate=0

```